app/models/index_manager.rb

```diff
@@ -9,11 +9,12 @@ def self.for(solr_url)
   def self.reindex!(solr_url: nil)
     solr_url ||= rebuild_solr_url
     manager = self.for(solr_url)
-    return if manager.in_progress?
+    return false if manager.in_progress?
```

Comment:
Seems fine, but if you're depending on this return value you should add a test for it.

Reply:
Whoop, I just missed it.